webapp/src/jaas/reducers/index.ts

```diff
@@ -0,0 +1,38 @@
+import {JaaSMeetingState, JaaSState} from 'jaas/util';
+import {combineReducers, AnyAction, Reducer} from 'redux';
+import {JaaSActionTypes} from '../action_types';
+import {StartMeetingWindowAction} from '../actions';
+
+type JaaSActions = StartMeetingWindowAction | AnyAction;
+
+function JaaSMeetingWindow(state: JaaSMeetingState = {}, action: JaaSActions) {
+    switch (action.type) {
+    case JaaSActionTypes.START_MEETING_WINDOW: {
+        return {
+            ...state,
+            jaasJwt: action.data.jaasJwt,
+            jaasRoom: action.data.jaasRoom
+        };
+    }
+    default:
+        break;
+    }
+    return state;
+}
+
+function openJaasMeetingWindow(state: any = null, action: AnyAction) {
+    switch (action.type) {
+    default:
+        break;
+    }
+    return state;
+}
```

Comment:
Is this used? Seems like the value never changes.

Reply:
Removed, kept only START_MEETING_WINDOW action related code, and moved the URLSearchParams code from loadInitialState to componentDidMount in jaas_conference.tsx